op-node/node/node.go

```diff
@@ -163,20 +164,21 @@ func (n *OpNode) initL2(ctx context.Context, cfg *Config, snapshotLog log.Logger
 
 func (n *OpNode) initRPCServer(ctx context.Context, cfg *Config) error {
 	var err error
```

Comment:
Can probably remove this error declaration here now?

Reply:
yes, good catch